examples/thread_test/src/main.rs

```diff
@@ -0,0 +1,230 @@
+#![feature(thread_local)]
+
+use std::cell::{Cell, RefCell};
+use std::collections::HashMap;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::thread;
+use std::thread::available_parallelism;
```

Comment:
Please don't import this function, but always reference the module (`thread::available_parallelism()`).

```suggestion
```